applications/luci-app-tor-hs/htdocs/luci-static/resources/view/tor_hs_common.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+'require view';
+'require dom';
+'require uci';
+'require fs';
+'require ui';
+'require rpc';
+'require form';
+'require tools.widgets as widgets';
+
+function setFlagBool(o) {
```

Comment:
Is this setFlagBool function really needed?
The same applies to the other files

Reply:
No. Removed.